src/main/java/org/jabref/gui/fieldeditors/PopOverUtil.java

```diff
@@ -0,0 +1,56 @@
+package org.jabref.gui.fieldeditors;
+
+import java.util.Optional;
+
+import javafx.scene.control.Button;
+import javafx.scene.control.ProgressIndicator;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.fieldeditors.journalinfo.JournalInfoView;
+import org.jabref.gui.util.BackgroundTask;
+import org.jabref.gui.util.TaskExecutor;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+import org.controlsfx.control.PopOver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PopOverUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PopOverUtil.class);
+
+    public static void showJournalInfo(Button button, BibEntry entry, DialogService dialogService, TaskExecutor taskExecutor) {
+        Optional<String> optionalIssn = entry.getField(StandardField.ISSN);
+
+        if (optionalIssn.isPresent()) {
+            PopOver popOver = new PopOver();
+            ProgressIndicator progressIndicator = new ProgressIndicator();
+            progressIndicator.setMaxSize(30, 30);
+            popOver.setContentNode(progressIndicator);
+            popOver.setDetachable(false);
+            popOver.setTitle(Localization.lang("Journal Information"));
+            popOver.setArrowLocation(PopOver.ArrowLocation.BOTTOM_CENTER);
+            popOver.setArrowSize(0);
+            popOver.show(button, 0);
+
+            BackgroundTask
+                    .wrap(() -> new JournalInfoView().populateJournalInformation(optionalIssn.get()))
+                    .onSuccess(updatedNode -> {
+                        popOver.setContentNode(updatedNode);
+                        popOver.show(button, 0);
+                    })
+                    .onFailure(exception -> {
+                        popOver.hide();
+                        String message = Localization.lang("Error while fetching journal information: %0",
+                                exception.getMessage());
+                        LOGGER.warn(message, exception);
+                        dialogService.notify(message);
+                    })
+                    .executeWith(taskExecutor);
+        } else {
+            dialogService.notify(Localization.lang("ISSN required for fetching journal information"));
```

Comment:
The api also accepts a journal name. I think its quite common that one has the journal title but not its issn, so I would be nice to cover this situation as well.

Reply:
Ah, nice. That means an "i" button also at the journal / journaltitle (BibLaTeX) field? This is nice!

Reply:
@tobiasdiez  Thanks, will add these changes

@koppor sorry is there another journal field other than the one in 'Required fields'?

<img width="1173" alt="image" src="https://github.com/JabRef/jabref/assets/12078268/0302bb1e-b1bb-4f12-93d3-ce26dd62fe15">